misc/config_tools/schema/checks/vm_types.xsd

```diff
@@ -7,7 +7,7 @@
 
   <xs:assert test="count(/acrn-config/vm[.//load_order = 'SERVICE_VM']) &lt; 2">
     <xs:annotation acrn:severity="error" acrn:report-on="//vm[.//load_order = 'SERVICE_VM']">
-      <xs:documentation>There can be at most one service VM, but more than one was configured. Verify there is only one VM with vm_type set to SERVICE_VM. </xs:documentation>
+      <xs:documentation>There may be at most one Service VM, but more than one was configured. Verify only one VM's VM type is set to SERVICE_VM. </xs:documentation>
```

Comment:
Would this error occur in the UI? If so, describe how to fix in UI?

Reply:
The VM load_order option is not visible in the UI so I suspect the only way this error would show up is if the schema file were manually edited and then the schema were loaded back into the configurator and saved again.  @junjiemao1 @NanlinXie What are the expectations on the configurator if a schema is read back in that contains an error such as this and there's no way for the developer to use the configurator to fix the problem?

Reply:
Generally, one can only fix a manually-corrupted scenario file by manually rolling back the change. I don't think the configurator can have a systematic way to fix those issues.

What the configurator may do is to refuse the loading of such XMLs.